pkg/db/migrations/2_checks.sql

```diff
@@ -22,11 +22,13 @@ CREATE TABLE checks(
 	description TEXT,
 	icon TEXT,
 	spec jsonb  NULL,
+	labels jsonb NULL,
 	owner text,
 	severity TEXT,
 	last_runtime TIMESTAMP,
 	next_runtime TIMESTAMP,
 	silenced_at TIMESTAMP NULL,
+	status boolean, -- status of last check executed
```

Comment:
Can we change this to a text as we could have different status e.g. warning, info etc..